package.json

```diff
@@ -27,7 +27,8 @@
     "redux-thunk": "^2.2.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "json-server --static ./build db.json",
+    "dev": "react-scripts start",
```

Comment:
@laurelherting `yarn start` is now reserved for production use. You'll need to do `yarn dev` in one terminal and `yarn run dev-server` in another terminal.